app.json

```diff
@@ -27,11 +28,18 @@
       "permissions": [
         "android.permission.RECORD_AUDIO",
         "android.permission.CAMERA"
-      ]
+      ],
+      "package": "com.quantumfitlabs.quantumfit"
```

Comment:
Did we create a new account for this?

Reply:
No, we did not, but I needed some name to run application on local android build